src/main/resources/config/application-local.properties

```diff
@@ -0,0 +1,17 @@
+aws.s3.accessKeyId=AKIAI7ACRR6VQEPFWMPQ
```

Comment:
Remove these configs. Anyone can read these.

Reply:
Yeah. It's a private repo so I committed for you guys.